api/src/pcapi/static/backoffice/js/addons/pc-batch-action-form.js

```diff
@@ -1,9 +1,6 @@
 /**
  * Add batch submit form to selection done using one `selectedRowsIds` within [PcTableMultiSelect](#pctablemultiselect).
- *
- * Form is renderer through a turbo frame which must be loaded with `loading="true"`.
- *
- * > Current integration does not yet support modal rendered with turbo `loading="lazy"`.
+ *.
```

Comment:
[Très optionnel] C'est vraiment juste pour montrer que j'ai relu :)
```suggestion
 *
```